src/platform/entity.ts

```diff
@@ -0,0 +1,51 @@
+import { IPFSv1, IPFSv2 } from '../misc'
+import { ContentMapping } from '../misc/content-mapping'
+import { generateValidator, JSONSchema, ValidateFunction } from '../validation'
+
+/**
+ * Non-exhaustive list of EntityTypes.
+ * @public
+ */
+export enum EntityType {
+  SCENE = 'scene',
+  PROFILE = 'profile',
+  WEARABLE = 'wearable',
+  STORE = 'store'
+}
+
+/**
+ * Internal representation of an entity in the catalyst.
+ *
+ * This Entity's content mappings adhere to ADR45.
+ *
+ * @public
+ */
+export type Entity = {
+  /** @deprecated ADR45 removed entity versions. */
+  version: string
+  id: IPFSv1 | IPFSv2
+  type: EntityType
+  pointers: string[]
+  timestamp: number
+  content: ContentMapping[]
```

Comment:
is it safe to change this field from optional to mandatory?

Reply:
I'd rather leave it optional as it is now, and then we can enforce it in the future if needed

Reply:
I have two questions:
- in which cases this is optional?
- why do we have Deployment and Entity as separated things in the catalyst? they share the same fields but with different names

Reply:
a) I dunno know. Just in the `catalyst-commons` is marked as optional. But if we think that it's always present (or should be), i'm in favor to mark it mandatory, and fix if something breaks. 
b) Which `Deployment` type do you refer, `DeploymentWithAuthChain`?

Reply:
a) I dunno know. Just in the `catalyst-commons` is marked as optional. But if we think that it's always present (or should be), i'm in favor to mark it mandatory, and fix if something breaks. 
b) Which `Deployment` type do you refer, `DeploymentWithAuthChain`?

Reply:
Nope, just Deployment

Reply:
This one https://github.com/decentraland/catalyst-commons/blob/master/src/types.ts#L88? 
Fortunately, it's only used in test as i see. We should delete it